packages/browser/src/index.ts

```diff
@@ -68,6 +70,34 @@ export default class LogtoClient {
     window.location.assign(signInUri);
   }
 
+  public async signOut(postLogoutRedirectUri?: string) {
+    if (!this.idToken || !this.oidcConfig) {
+      return;
```

Comment:
maybe throw an error?

Reply:
browser package doesn't have error handling utilities at this point, will throw error once #175 is merged.

Reply:
should to await oidc config instead of directly returning.